src/commands/iTunes.js

```diff
@@ -0,0 +1,64 @@
+const { Command } = require('@dyno.gg/dyno-core');
+const superagent = require('superagent');
+
+class iTunes extends Command {
+	constructor(...args) {
+		super(...args);
+
+		this.aliases      = ['iTunes'];
+		this.module       = 'Fun';
+		this.description  = 'Get info on a song.';
+		this.usage        = 'itunes [song name]';
+		this.example      = 'itunes Perfect Duet';
+		this.cooldown     = 3000;
+		this.expectedArgs = 1;
+	}
+
+	async execute({ message, args }) {
+        args = args.join(' ');
+	    try {
+            let res = await superagent.get('https://itunes.apple.com/search').query({'term': args}).query({'media':'music'});
```

Comment:
No quotes around the quey keys, and spaces like `{ term: args }`, also does query not support `{ term: args, media: 'music' }` ?